src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -319,7 +319,6 @@ private void SendViewAddedEventToWindow(IntPtr data)
                 {
                     this.HoveredSignal().Disconnect(hoverEventCallback);
                 }
-
```

Comment:
can you remove this change? this file must not be included this patch.

Reply:
Thank you I fixed.